src/nvim/menu.c

```diff
@@ -1294,34 +1294,34 @@ static char_u *popup_mode_name(char *name, int idx)
 ///             allocated.
 ///
 /// @return a pointer to allocated memory.
-static char_u *menu_text(const char_u *str, int *mnemonic, char_u **actext)
+static char *menu_text(const char *str, int *mnemonic, char **actext)
   FUNC_ATTR_NONNULL_RET FUNC_ATTR_WARN_UNUSED_RESULT
   FUNC_ATTR_NONNULL_ARG(1)
 {
-  char_u *p;
-  char_u *text;
+  char *p;
+  char *text;
 
   // Locate accelerator text, after the first TAB
-  p = vim_strchr(str, TAB);
+  p = (char *)vim_strchr((char_u *)str, TAB);
   if (p != NULL) {
     if (actext != NULL) {
-      *actext = vim_strsave(p + 1);
+      *actext = (char *)vim_strsave((char_u *)p + 1);
     }
     assert(p >= str);
-    text = vim_strnsave(str, (size_t)(p - str));
+    text = (char *)vim_strnsave((char_u *)str, (size_t)(p - str));
   } else {
-    text = vim_strsave(str);
+    text = (char *)vim_strsave((char_u *)str);
```

Comment:
this should just be `xstrdup`

Reply:
Yeah, figured. I'll fix the other vim_strsave as well :+1: